docs/develop/architecture.rst

```diff
@@ -16,6 +16,9 @@ As any other Django project Portal organizes its functionality in several apps:
   requests to a community, removing and inviting users to become members of a
   community.
 * **users** - showing and editing user personal profile.
+* **meetup** - handles meetup locations and meetup functionality, like adding, showing, editing,
```

Comment:
Make it shorter - "handles meetup locations and meetup functionality" - that will probably we enough as an overview.